doc/release/1.14.0-notes.rst

```diff
@@ -108,6 +108,12 @@ the result is always a view on the original masked array.
 This breaks any code that used ``masked_arr.squeeze() is np.ma.masked``, but
 fixes code that writes to the result of `.squeeze()`.
 
+Renamed first parameter of ``can_cast`` from ``from`` to ``from_``
```

Comment:
Is it correct to add this to the compatibility notes?

Reply:
Yep.